bec_alerts/jinja2/healthcheck.txt

```diff
@@ -0,0 +1,10 @@
+A healthcheck event was received in the bec-alerts queue. It worked!
+
+{{ issue.message }}
+{%- for frame in issue.stack_frames|reverse %}
+    at {{ frame.function }}({{ frame.module }}:{{ frame.lineNo|default('', True) }}:{{ frame.colNo|default('', True) }})
+{%- endfor %}
+
+This issue can be viewed in Sentry:
+
+https://sentry.prod.mozaws.net/operations/nightly-js-errors/issues/{{ issue.groupId }}/
```

Comment:
I wonder if there should be a "generic" message template that you can use for things. Just thinking out loud--no need to do anything here.

Reply:
I was thinking of making the stack trace bits into a macro. Maybe the link-generation too.

Sounds like good steps for the third template.